deb-get

```diff
@@ -699,6 +699,28 @@ function deb_code() {
     SUMMARY="Code editing. Redefined."
 }
 
+function deb_nala() {
+    ARCHS_SUPPORTED="amd64 arm64 armhf i386"
+    GPG_KEY_URL="https://deb.volian.org/volian/scar.key"
+    APT_LIST_NAME="nala"
+    APT_REPO_URL="deb  http://deb.volian.org/volian/ scar main"
```

Comment:
The new format does not have the leading `deb`.

Reply:
oops sorry: missed that. Sorted